### ISSUE TITLE
'libRblas.so: No such file or directory' during package installation

### ISSUE THREAD

**Issue by jay-sf** [SteffenMoritz/imputeTS#63]
Hi,

first of all thanks for maintaining this great package!

On Ubuntu 22.04 LTS with a freshly installed R using the regular `sudo apt install r-base-core` I tried to `install.packages('imputeTS')` and received this error:

```
Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/home/jay/R/x86_64-pc-linux-gnu-library/4.2/fracdiff/libs/fracdiff.so':
  libRblas.so: cannot open shared object file: No such file or directory
Calls: <Anonymous> ... asNamespace -> loadNamespace -> library.dynam -> dyn.load
Execution halted
```
The `sessionInfo()`, though, shows that BLAS and LAPACK actually are find OpenBLAS which I think is what `libRblas.so` is referring to.

```
R version 4.2.1 (2022-06-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 22.04 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.20.so
```
I elaborate on this a little further in [this Stack Overflow question](https://stackoverflow.com/q/72966093/6574038). There is an answer that suggests a package bug which is why I'm opening a ticket here.

Thanks for any help!

All the best

**Comment by SteffenMoritz** [comment]
Hello jay, thanks for reporting your issue!
(the world needs people reporting issues to keep the package quality up 👍 )

Does the following work:
`install.packages('fracdiff')`

And if this works, please try installing imputeTS afterwards again :)

From the error message it seems like the error is rather related to fracdiff, which is rather deep in the dependency tree (imputeTS imports the forcast package, which imports fracdiff).

So I would assume some combination from fracdiff and some weird local installation/configuration things ... which would be nothing, with which I could help.

But would be great if we could indeed make sure this issue is not directly related to imputeTS code.

**Comment by jay-sf** [comment]
Thanks a ton, Steffen! 

Your comment brought me on the right track, I installed `fracdiff` as suggested and tried to install `imputeTS` thereafter. The error occurred three more times with `quadprog`, `tseries`, `forecast`. So I suspected right that I had fabricated some clash while switching from locally compiled R to R installed from an `apt` repository. Since I could assume that this would happen again and again, the best thing was to purge R and descendants completely from the system and reinstall everything, which appears to have worked well! Also `imputeTS` is back :)

So that those who run into similar problems can benefit from it I put the how-to together in my [answer on Stack Overflow](https://stackoverflow.com/a/72978650/6574038).

All the best, J
